lib/Basic/Unix/TaskQueue.inc

```diff
@@ -249,45 +247,48 @@ bool Task::execute() {
   return false;
 }
 
-static bool readFromAPipe(int Pipe, std::string &Output,
-                          UnifiedStatsReporter *Stats,
-                          bool UntilEnd) {
+/// \p Pipe must be in blocking mode, and must contain unread data.
+/// Read the data in \p Pipe, and append it to \p Output.
```

Comment:
Nitpick: If you're going to give this a doc comment, please put this line first, and possibly in its own paragraph, so that it works better with Doxygen and Quick Help.

Reply:
Yup! Love those nitpicks!